src/pybind/mgr/dashboard/settings.py

```diff
@@ -158,12 +160,16 @@ def py2ceph(pytype):
                 'perm': 'r'
             })
         elif cmd.startswith('dashboard set'):
-            cmd_list.append({
+            cmd_entry = {
                 'cmd': '{} name=value,type={}'
                        .format(cmd, py2ceph(opt['type'])),
                 'desc': 'Set the {} option value'.format(opt['name']),
                 'perm': 'w'
-            })
+            }
+            if handles_secret(cmd):
+                cmd_entry['cmd'] = cmd
+                cmd_entry['desc'] = '{} read from -i <file>'.format(cmd_entry['desc'])
+            cmd_list.append(cmd_entry)
```

Comment:
I'm a bit surprised that there's nothing in the command specifications to indicate that one param is read from a file... I checked `src/mon/MonCommand.h` but I couldn't find any.

@neha-ojha do you know if there any way to make this `-i <file>` input argument somehow explicit? I see that some commands document it (`auth import`, `osd new`), but no other indication (flag, cephtype...).